engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -26,6 +26,9 @@
  * TODO: Add javadocs
  */
 public final class SetWireframe implements StateChange {
+    public static final int ENABLED = GL_LINE;
+    public static final int DISABLED = GL_FILL;
+
```

Comment:
Let me remove these variables for avoiding any possible confusions and move `mode` selection decision-making to `SetWireframeTask` (task receives a boolean and selects the correct OpenGL variable.)